### PR TITLE
Update linux workshop based on review

### PR DESCRIPTION
**Pull request by katiewasnothere**
This PR updates the linux basics workshop with the changes discussed in a previous workshop review. Note: some of the changes include adding new explanations/changing sentences and review of them would be greatly appreciated. 

Signed-off-by: Kathryn Baldauf <kabaldau@microsoft.com>

### CODE REVIEW

File: content/english/linux-basics/_index.md [hunk 1]
```diff
@@ -10,6 +10,18 @@ icon: "fas fa-microchip"
...
+- A computer that is *not* a Chromebook 
+- 3GB of free space on your computer
+- ~10 minutes to download VirtualBox
+- ~30 minutes to download the virtual machine image
+
+{{% /notice %}}
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zfdlPZYlgtk" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
 # Welcome
```
Comment (ozhang22) [inline content/english/linux-basics/_index.md:25]:
```suggestion
## Welcome
```

For accessibility

File: content/english/linux-basics/activity-2.md [hunk 2]
```diff
@@ -7,9 +7,32 @@ difficulty: "Intermediate"
...
+
+A filesystem is a structured collection of files and data on a computer disk or drive. The highest level of a filesystem is referred to as the "root". You can think of it as the beginning of the filesystem. 
+
+{{% notice tip %}}
+Fun fact: filesystems are often described as a "tree". This is because the filesystem looks like an upside down tree with each folder as a branch or leaf. For example, your filesystem may look like the below.
+
+![picture of a filesystem tree](../images/filesystem-tree.png?classes=border,shadow)
+{{% /notice %}}
+
+For example, on a windows computer, you can browse the computer's "C:\" drive filesystem in the "file explorer" by navigating to the below
```
Comment (ozhang22) [inline content/english/linux-basics/activity-2.md:24]:
```suggestion
On a Windows computer, you can browse the computer's "C:\" drive filesystem in the "file explorer" like below
```

File: content/english/linux-basics/activity-2.md [hunk 3]
```diff
@@ -7,9 +7,32 @@ difficulty: "Intermediate"
...
+
+![picture of a windows file explorer](../images/filesystem-windows.png?classes=border,shadow)
+
+On a mac computer, you can browse the computer's home filesystem in "finder" like below
```
Comment (ozhang22) [inline content/english/linux-basics/activity-2.md:28]:
```suggestion
On a Mac computer, you can browse the computer's home filesystem in "Finder" like below
```

File: content/english/linux-basics/activity-2.md (hunk 2 above)
Comment (ozhang22) [reply content/english/linux-basics/activity-2.md:24]: Do we need to talk about what the C:\ drive is?

File: content/english/linux-basics/activity-5.md [hunk 4]
```diff
@@ -7,9 +7,29 @@ difficulty: "Intermediate"
...
 *Oh no. Some of our files might have been messed with! Let’s secure our notes, file1. We don't want a hacker to mess with these!*
 
-### But, How do we know how secure our files are?
+### How do we limit access to our files? 
+
+By giving users "permissions" on files! 
+
+Permissions are a way for the computer to determine who has access to files and what they can do with those files.
+
+In linux, there are three common types of access to a file that a user might request: "read", "write", or "execute".
```
Comment (ozhang22) [inline content/english/linux-basics/activity-5.md:20]:
```suggestion
In Linux, there are three common types of access to a file that a user might request: "read", "write", or "execute". 
```

File: content/english/linux-basics/activity-7.md [hunk 5]
```diff
@@ -47,10 +49,10 @@ Sometimes looking at the first and last lines of the file allow us to realize we
...
 
 `cat` shows the contents of a file.  
-The format is: `cat [flags] [Filename]`
+The format is: `cat [filename]`
 
 {{% notice tip %}} 
-Less and more are commands similar to cat.
+`less` and `more` are commands similar to `cat`.
 
 #### MORE
```
Comment (ozhang22) [inline content/english/linux-basics/activity-7.md:57]:
```suggestion
#### The more command
```

File: content/english/linux-basics/activity-7.md [hunk 6]
```diff
@@ -61,7 +63,7 @@ The format is: `more [filename]`
 
 #### LESS
```
Comment (ozhang22) [inline content/english/linux-basics/activity-7.md:64]:
```suggestion
#### The less command
```

File: content/english/linux-basics/activity-5.md [hunk 7]
```diff
@@ -63,4 +82,3 @@ What permissions do the other two values give?
 | 2 | write |
```
Comment (ozhang22) [inline content/english/linux-basics/activity-5.md:82]: I don't like the explanation that 7 is a combination of 4, 2, 1. Might be better to just fully enumerate what 3, 5, 6, 7, all mean...

File: content/english/linux-basics/activity-2.md [hunk 8]
```diff
@@ -7,9 +7,32 @@ difficulty: "Intermediate"
...
+
+{{% notice tip %}}
+Fun fact: filesystems are often described as a "tree". This is because the filesystem looks like an upside down tree with each folder as a branch or leaf. For example, your filesystem may look like the below.
+
+![picture of a filesystem tree](../images/filesystem-tree.png?classes=border,shadow)
+{{% /notice %}}
+
+For example, on a windows computer, you can browse the computer's "C:\" drive filesystem in the "file explorer" by navigating to the below
+
+![picture of a windows file explorer](../images/filesystem-windows.png?classes=border,shadow)
```
Comment (ozhang22) [inline content/english/linux-basics/activity-2.md:26]:
```suggestion
![Picture of a Windows file explorer](../images/filesystem-windows.png?classes=border,shadow)
```

File: content/english/linux-basics/activity-2.md [hunk 9]
```diff
@@ -7,9 +7,32 @@ difficulty: "Intermediate"
...
+
+On a mac computer, you can browse the computer's home filesystem in "finder" like below
+
+![picture of the mac file finder](../images/home-folder-mac.jpg?classes=border,shadow)
```
Comment (ozhang22) [inline content/english/linux-basics/activity-2.md:30]:
```suggestion
![Picture of the Mac file finder](../images/home-folder-mac.jpg?classes=border,shadow)
```

File: content/english/linux-basics/activity-3.md [hunk 10]
```diff
@@ -7,20 +7,30 @@ difficulty: "Intermediate"
...
+![ls command](../images/03_ls-command.png?classes=border,shadow)
+
+### How do I get more info about the files in this directory?
+
 Try it:
 ```
 ls -l
 ```
 
 ![ls-l command](../images/03_ls-l.png?classes=border,shadow)
```
Comment (ozhang22) [inline content/english/linux-basics/activity-3.md:31]:
```suggestion
![ls -l command](../images/03_ls-l.png?classes=border,shadow)
```

File: content/english/linux-basics/activity-4.md [hunk 11]
```diff
@@ -24,8 +25,43 @@ mkdir files
...
+
+![ls-l command](../images/03_ls-l.png?classes=border,shadow)
+
+*Great job, you now know how to move files around. We will use this knowledge soon to recover from the hack.*
+
 ### What if I want to create a new file?
-To create a file we can use the `touch` command. The tough command creates an empty file with no data. 
+To create a file we can use the `touch` command. The touch command creates an empty file with no data. 
 
 Try the command below to create a file named `file1`:
```
Comment (ozhang22) [inline content/english/linux-basics/activity-4.md:67]: Below:

An empty file **doesn't** have much use, so **let's** add some data to it, like our name.

File: content/english/linux-basics/activity-4.md [hunk 12]
```diff
@@ -50,12 +86,22 @@ vim file1
 ![ls-l command](../images/04_vim.PNG?classes=border,shadow)
 
 We are now in the Vim editor. Press `i` to enter insert mode. Now type in `nuvi` and press esc to exit insert mode. Finally type in `:wq` and press enter The colon tells vim you are entering a command. The w and q then tell vim you want to write the changes you made and quit.
```
Comment (ozhang22) [inline content/english/linux-basics/activity-4.md:88]:
```suggestion
We are now in the Vim editor. Press `i` to enter insert mode. Now type in `nuvi` and press `esc` to exit insert mode. Finally type in `:wq` and press enter. The colon tells vim you are entering a command. The w and q then tell Vim you want to write the changes you made and quit.
```

File: content/english/linux-basics/activity-6.md [hunk 13]
```diff
@@ -7,18 +7,15 @@ difficulty: "Intermediate"
...
-`grep` is a command used to search for a pattern of `chars`. There are different flags or options that can change how it is used.  
+`grep` is a command used to search for text within files. There are different flags or options that can change how it is used.  
 The format is: `grep [flag] “[pattern]” [filename]`.
 
-{{% notice tip %}}
-A `char` is any letter, symbol, or number. Examples include `!`, `h`, `2`, etc.
-{{% /notice %}}
-
-
 |flags | Uses |
```
Comment (ozhang22) [inline content/english/linux-basics/activity-6.md:19]:
```suggestion
| flags | Uses |
```

File: content/english/linux-basics/activity-8.md [hunk 14]
```diff
@@ -7,6 +7,8 @@ difficulty: "Intermediate"
 weight: 11
```
Comment (ozhang22) [inline content/english/linux-basics/activity-8.md:7]: Remove the period at the end of the description and prereqs for all files.

File: content/english/linux-basics/activity-8.md [hunk 15]
```diff
@@ -7,6 +7,8 @@ difficulty: "Intermediate"
...
 ---
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/fAI1kyAoVTA" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
 *Let's look at the secret message to make sure this weird `malware.sh` file hasn't done anything to it!*
 
 ### Do we know a command that can help us find the location of the `secret-message.txt` file?
```
Comment (ozhang22) [inline content/english/linux-basics/activity-8.md:14]: Below:

**Let's** move into that directory and check

File: content/english/linux-basics/activity-8.md [hunk 16]
```diff
@@ -28,20 +30,21 @@ ls -la
...
+Fun Fact: Running `diff --help` will output information on how to use the `diff` tool, including what options can be used with the tool. 
 {{% /notice %}}
 
 Check the difference between the two files. How many differences did you find? Put it in the file you made!
 
 {{% notice tip %}}
+
 ### A better way to see the differences
 
 `vimdiff` is a command that opens a window with the files in separate tabs highlighting the differences.
```
Comment (ozhang22) [inline content/english/linux-basics/activity-8.md:50]: Are we able to get a screenshot for this?

File: content/english/linux-basics/vbox-setup.md [hunk 17]
```diff
@@ -7,15 +7,17 @@ difficulty: "Intermediate"
...
 ---
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zoutwedSLKI" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
 *So you can help us, we are providing you with a copy of the hacked computer as a virtual machine.*
 
 {{% notice info %}}
-## What is a virtual machine
+### What is a virtual machine?
 A **virtual machine** (commonly called a vm) is a computer file (called an `image`), that behaves like an actual computer.
```
Comment (ozhang22) [inline content/english/linux-basics/vbox-setup.md:16]:
```suggestion
A **virtual machine** (commonly called a VM) is a computer file (called an `image`), that behaves like an actual computer.  
```

File: content/english/linux-basics/vbox-setup.md [hunk 18]
```diff
@@ -7,15 +7,17 @@ difficulty: "Intermediate"
...
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zoutwedSLKI" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
 *So you can help us, we are providing you with a copy of the hacked computer as a virtual machine.*
 
 {{% notice info %}}
-## What is a virtual machine
+### What is a virtual machine?
 A **virtual machine** (commonly called a vm) is a computer file (called an `image`), that behaves like an actual computer.  
 In other words, creating a computer within a computer. It runs in a window, much like any other program.  
 This gives you the same experience on a vm as you would on a regular computer.
```
Comment (ozhang22) [inline content/english/linux-basics/vbox-setup.md:18]:
```suggestion
This gives you the same experience on a VM as you would on a regular computer.
```

File: content/english/linux-basics/vbox-setup.md [hunk 19]
```diff
@@ -7,15 +7,17 @@ difficulty: "Intermediate"
...
 
-The vm is `sandboxed` from the rest of the system, meaning that the software inside a vm can’t effect the computer itself.  
+The vm is `sandboxed` from the rest of the system, meaning that the software inside a vm can’t affect the computer itself.
```
Comment (ozhang22) [inline content/english/linux-basics/vbox-setup.md:20]:
```suggestion
The VM is `sandboxed` from the rest of the system, meaning that the software inside a VM can’t affect the computer itself.  
```

File: content/english/linux-basics/vm-import-setup.md [hunk 20]
```diff
@@ -7,10 +7,12 @@ difficulty: "Intermediate"
 weight: 2
 ---
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/ffcyyJXEhwY" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
 *Good work on getting the host environment ready! Now let's load the image of the hacked computer as a vm.*
```
Comment (ozhang22) [inline content/english/linux-basics/vm-import-setup.md:12]:
```suggestion
*Good work on getting the host environment ready! Now let's load the image of the hacked computer as a VM.*
```

File: content/english/linux-basics/vm-import-setup.md [hunk 21]
```diff
@@ -28,7 +30,7 @@ VirtualBox uses the .ova for their machine images.
 Let's download our copy of the computer! Click the link below to download.
```
Comment (ozhang22) [inline content/english/linux-basics/vm-import-setup.md:30]: Above:

**Image** files make it easy to send exact copies of computers.

File: content/english/linux-basics/launch-vm.md [hunk 22]
```diff
@@ -7,6 +7,8 @@ difficulty: "Intermediate"
 weight: 3
 ---
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/eqHr7UeKs70" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
 *Now that everything is installed, let's launch the virtual machine!*
 
 ### Starting the vm
```
Comment (ozhang22) [inline content/english/linux-basics/launch-vm.md:14]:
```suggestion
### Starting the VM
```

**Review by ozhang22 — APPROVED** [review]

File: content/english/linux-basics/activity-2.md (hunk 2 above)
Comment (katiewasnothere) [reply content/english/linux-basics/activity-2.md:24]: As in talking about what a drive is or as in talking about the significance of the C:\ drive itself? If the former maybe it's best to take out the word "driver". Otherwise, if we want to explain the significance, then maybe we could add that as like a fun fact or something?

File: content/english/linux-basics/activity-8.md (hunk 14 above)
Comment (katiewasnothere) [reply content/english/linux-basics/activity-8.md:7]: Should I get rid of difficulty as well?

File: content/english/linux-basics/activity-8.md (hunk 16 above)
Comment (katiewasnothere) [reply content/english/linux-basics/activity-8.md:50]: Oh I must have missed this one. Should I try to get it?
